features/kiesrecht.feature

```diff
@@ -12,28 +12,20 @@ Rule: Aanduiding uitgesloten kiesrecht (38.10) wordt geleverd als boolean
 	- waarde "A" wordt geleverd als boolean waarde true
 
 	@gba
-	Abstract Scenario: Aanduiding uitgesloten kiesrecht (38.10) <uitsluiting kiesrecht> vertaald naar boolean
+	Scenario: Aanduiding uitgesloten kiesrecht (38.10) "A" vertaald naar boolean true
 	  Gegeven het systeem heeft een persoon met de volgende gegevens
 		| naam                        | waarde     |
 		| burgerservicenummer         | 999990001  |
 		En de persoon heeft de volgende kiesrecht gegevens
-		| aanduiding uitgesloten kiesrecht (38.10) | <uitgesloten kiesrecht> |
-		| einddatum uitsluiting kiesrecht (38.20)  | 20300101                |
+		| aanduiding uitgesloten kiesrecht (38.10) | A |
```

Comment:
Zie opmerkingen in de gba-dev scenario

Reply:
rij met naam en waarde toegevoegd (neem aan dat je die opmerking bedoelt)